agent/bench-scripts/postprocess/process-iteration-samples

```diff
@@ -209,9 +216,13 @@ foreach $sample (@json_samples) { # samples 0..N
 						my %sample = (get_label('uid_label') => $sample_hash{$metric_type}{$metric_name}[$i]{get_label('uid_label')});
 						unshift( @{ $iteration{$metric_type}{$metric_name} }, \%sample);
 						$iteration_matching_index = 0;
+						#print "NEW $mapped_uid -- [$metric_type][$metric_name][$i] ==> [$metric_type][$metric_name][$iteration_matching_index]\n";
+					} else {
+						#print "EXISTING $mapped_uid -- [$metric_type][$metric_name][$i] ==> [$metric_type][$metric_name][$iteration_matching_index]\n";
```

Comment:
Yes, diagnostic code I left in because I was too scared to take it out for fear that I had introduced another bug.

I'll post some test RPMs for people to try first before I remove.

Reply:
Will address this is PR #2071.